<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bial Batch Segmentation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonPrevious">
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButtonNext">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="GraphicsView" name="graphicsView"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButtonReset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_folder"/>
    <addaction name="separator"/>
    <addaction name="actionSet_default_folder"/>
    <addaction name="actionSet_result_folder"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionStart_Segmentation"/>
    <addaction name="actionReset_seeds"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_folder">
   <property name="text">
    <string>Open folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSet_default_folder">
   <property name="text">
    <string>Set default folder</string>
   </property>
  </action>
  <action name="actionSet_result_folder">
   <property name="text">
    <string>Set result folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionStart_Segmentation">
   <property name="text">
    <string>Start Segmentation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionReset_seeds">
   <property name="text">
    <string>Reset seeds</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
